.gitignore

```diff
@@ -1,36 +1 @@
-# Rust build directory
 **/target
```

Comment:
Since we're in a workspace now, just the root should be sufficient, right?

Reply:
you mean, just `target`?